src/ScottPlot5/ScottPlot/Range.cs

```diff
@@ -47,9 +52,9 @@ public Range(double min, double max)
         /// <returns>The normalized value</returns>
         public double Normalize(double value, bool clamp = false)
         {
-            if (Max == Min) // When Min == Max we would divide by zero without this check.
+            if (Max == Min)
             {
-                return Min;
+                throw new ArgumentException("Cannot normalize the value to a range of zero");
```

Comment:
If we allow this to throw we need to check for `Min == Max` in `GetArgbValues` (or potentially in Colormap.GetColor), or we're going to crash on some heatmaps.